docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -128,3 +128,12 @@ _Available in private preview in Grafana Cloud_
 The Grafana Assume Role authentication provider lets Grafana Cloud users of the CloudWatch data source authenticate with AWS without having to create and maintain long term AWS Users. Using the new assume role authentication method, you no longer have to rotate access and secret keys in your CloudWatch data source. Instead, Grafana Cloud users can create an identity access and management (IAM) role that has a trust relationship with Grafana's AWS account; Grafana's AWS account will then use AWS Secure Token Service (STS) to create temporary credentials to access the user's AWS data.
 
 To learn more, refer to the [CloudWatch authentication documentation](/docs/grafana/next/datasources/aws-cloudwatch/aws-authentication).
+
+## Permission validation on custom role creation and update
+
+<!-- Mihaly Gyongyosi -->
+<!-- Enterprise -->
+
```

Comment:
The first is just a note for me. We'd like to add a release date to these entries for the Cloud version. Can you add that in and then commit this change?
```suggestion
<!-- already in on-prem -->

<Date>

```

Reply:
I don't know the exact release date, because it started to roll out to different release channels after I merged it (https://github.com/grafana/grafana/pull/73804), so I put the PR's merge date.